### ISSUE TITLE
Is it possible to add more menus of levels?

### ISSUE THREAD

**Issue by LiamKarlMitchell** [ryanmcnz/maplunky#6]
This would let people create new packs without over-writing existing.

**Comment by ryanmcnz** [comment]
Yeah should be possible. Just need to implement importing and exporting entire packs of maps, giving each pack a custom name, etc.
